### PR TITLE
Task/cardpart pillview

### PR DESCRIPTION
**Pull request by badrinathvm**
CardPartPillLabel provides you the rounded corners, text aligned being at the center along with vertical and horizontal padding capability.

image

### CODE REVIEW

File: Example/CardParts/CardPartPillLabelCardController.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
+//
+//  CardPartPillLabelCardController.swift
+//  CardParts_Example
+//
+//  Created by Venkatnarayansetty, Badarinath on 8/2/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```
Comment (lwdupont) [inline Example/CardParts/CardPartPillLabelCardController.swift]: Change the copyright?

**Comment by croossin** [comment]
Let's add in example app a demo of how to use it with RxSwift

File: Example/CardParts/MainViewController.swift [hunk 2]
```diff
@@ -33,7 +33,8 @@ class MainViewController: CardsViewController {
             CardPartCenteredViewCardController(), // Centered card part
             ReactiveCardController(), // Demo RxSwift
             StateCardController(), // Demo states
-            CardPartBorderViewController() // Border Cards
+            CardPartBorderViewController(), // Border Cards
+            CardPartPillLabelCardController() //Pill label
```
Comment (croossin) [inline Example/CardParts/MainViewController.swift]: Little thing, but add a space after `//` 😉

**Comment by badrinathvm** [comment]
Adopted RX Swift

File: Example/CardParts/MainViewController.swift (hunk 2 above)
Comment (badrinathvm) [reply Example/CardParts/MainViewController.swift]: done.

File: Example/CardParts/CardPartPillLabelCardController.swift [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        stackView.spacing = 8
+        stackView.distribution = .equalSpacing
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 10)
+        
+        [label1, label2, label3].forEach { label in
+            stackView.addArrangedSubview(label)
+        }
+        
+        viewModel.labelText.asObservable().bind(to: label3.rx.labelText).disposed(by: bag)
```
Comment (croossin) [inline Example/CardParts/CardPartPillLabelCardController.swift]: Awesome!

File: Example/CardParts/CardPartPillLabelCardController.swift (hunk 1 above)
Comment (badrinathvm) [reply Example/CardParts/CardPartPillLabelCardController.swift]: done.

**Pull request merged**